app/scripts/modules/plugins/tsconfig.json

```diff
@@ -0,0 +1,38 @@
+{
```

Comment:
does this mean plugins must be typescript?

Reply:
nope, i just used typescript to be like the rest of deck